docs/deployment-arbitrary-cluster.md

```diff
@@ -0,0 +1,50 @@
+# Deployment on Arbitrary Cluster
+
+In this guide, we will cover the installation of NetworkManager
+and kubernetes-nmstate on your arbitrary cluster.
+
+## Requirements
+
+This guide requires you to have your own Kubernetes/OpenShift cluster. If you
+don't have one and just want to try kubernetes-nmstate out, please refer to
+the [deployment on local cluster](deployment-local-cluster.md) guide.
+
+In order to get kubernetes-nmstate running, NetworkManager
+must be installed on the node.
+
+### NetworkManager
```

Comment:
Can we add one sentence about what it network manager?

Reply:
Done